worldengine/cli/main.py

```diff
@@ -313,6 +314,14 @@ def main():
                              "If not provided, the default values will be used. \n" +
                             "[default = .059/.222/.493/.764/.927/.986/.998]",
                             metavar="#/#/#/#/#/#/#")
+    g_generate.add_argument('-gv', '--gamma-value', dest='gv', type=float,
```

Comment:
Yes. When I hear "gamma value" the first thing I think of is something related to the brightness of the image-output.
I would maybe like to see a link to some page that explains this in more detail.